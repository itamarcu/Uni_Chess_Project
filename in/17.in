load in/16.xml
start
undo
undo
move <1,G> to <3,F>
quit
