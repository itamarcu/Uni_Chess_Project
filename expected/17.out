 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Starting game...
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ m _ _ _ _ _ |
3| _ _ _ _ _ _ _ m |
2| m m _ n m m m _ |
1| r _ b N k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Empty history, no move to undo
Enter your move (white player):
Empty history, no move to undo
Enter your move (white player):
8| R N B Q K B _ R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ m _ _ _ _ _ |
3| _ _ _ _ _ n _ m |
2| m m _ n m m m _ |
1| r _ b N k b _ r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Exiting...
